elliottlib/bzutil.py

```diff
@@ -177,8 +177,22 @@ def get_remotelinks(self, bugid):
                 linkdict[link.relationship] = link.object.url
         return linkdict
 
-    def update_bug(self, bugid, status):
-        self._client.transition_issue(self._client.issue(bugid), status)
+    def update_bug_status(self, bugid, target_status, comment_for_release=None, noop=False):
+        current_state = self._client.issue(bugid).fields.status.name
+        if noop:
+            logger.info(f"Would have changed BZ#{bugid} from {current_state} to {target_status}")
+            return
+        self._client.transition_issue(self._client.issue(bugid), target_status)
+        comment = f'Elliott changed bug status from {current_state} to {target_status}.'
+        self.add_comment_to_bug(self, bugid, comment, True, comment_for_release)
```

Comment:
I would like this to have a single optional param "comment" for custom comments, and then we can have another param called "status_update_comment=True" for the standard comment message. 

And then let's make add_comment_to_bug a generic/dumb method which will just add a comment string passed to it. Then we have 2 calls to comment_to_bug from this method